codechecker_lib/database_handler.py

```diff
@@ -96,6 +97,9 @@ def _create_or_update_schema(self, use_migration=True):
         except sqlalchemy.exc.SQLAlchemyError as alch_err:
             LOG.error(str(alch_err))
             sys.exit(1)
+        except CommandError as cerr:
+            LOG.error(str(cerr))
```

Comment:
Based on the code review feedback we should provide more user friendly error messages not just printing out the message from the exception. These error messages will be read by our users who might not be familiar with SQLAlchemy or Alembic errors.
I try to improve it in any new improvement and update our existing source too.